src/bindings/python/fluxacct/accounting/test/test_bank_subcommands.py

```diff
@@ -25,15 +24,17 @@ def setUpClass(self):
         # create example accounting database
         c.create_db("TestBankSubcommands.db")
         global acct_conn
+        global cur
         acct_conn = sqlite3.connect("TestBankSubcommands.db")
+        cur = acct_conn.cursor()
```

Comment:
Good suggestion, thanks. I can wrap this `connect` call in a try/except block, similar to my approach in the front-end script `flux-account.py`. If for some reason it is unable to connect to the SQLite file, it will raise a `sqlite3.OperationalError` and I can just exit the program with an error message saying it was unable to connect to the file. 

To the best of my knowledge, if the connection is made successfully with `sqlite3.connect()`, then calling creating a `Cursor` object or using its methods (like `.fetchall()`, `.fetchone()`, or `.execute()`) will not raise an exception. [Here](https://docs.python.org/3/library/sqlite3.html#sqlite3.Cursor) is the documentation I referenced.

For example, if a `SELECT` statement were to return no results, then calling `.fetchone()` would return `None` and can be handled on its own. I hope this explanation makes sense.